src/pal/tools/cppmunge/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ project(cppmunge)
 
 add_definitions(-D_BSD_SOURCE) # Needed to get realpath part of CRT for c99 compat.
 
+if(CLR_CMAKE_PLATFORM_LINUX)
+  add_definitions(-D__LINUX__=1)
```

Comment:
For reference, here's an example of use in LLDB:
https://github.com/llvm-mirror/lldb/blob/master/source/Target/Platform.cpp#L572